be/src/exec/workgroup/scan_executor.cpp

```diff
@@ -28,7 +29,7 @@ ScanExecutor::~ScanExecutor() {
 void ScanExecutor::initialize(int num_threads) {
     _num_threads_setter.set_actual_num(num_threads);
     for (auto i = 0; i < num_threads; ++i) {
-        _thread_pool->submit_func([this]() { this->worker_thread(); });
+        _thread_pool->submit_func([this]() { this->worker_thread(); }, ThreadPool::HIGH_PRIORITY);
```

Comment:
Is threadpool's underlying queue priority queue?  submit should keep non-blocking mode(try_submit)

Reply:
1. it's not, but a vanilla `std::deque`
2. it's non-blocking. the return status indicates submitted or not